docs/dev/notebooks.md

```diff
@@ -37,7 +48,7 @@ Thus, for notebooks with external dependencies, **all cells must have their outp
 
 ## Lifecycle 
 
-Notebooks are handled differently based on whether they rely on features in the pre-release build of cirq or not. 
+We should follow different configurations for notebooks depending on whether they rely on features in the pre-release build of cirq or not. 
```

Comment:
"We should..." used here still gives the sense that these steps are applied by someone other than the notebook author. Is this the case? Otherwise, I'd recommend "You should..." instead.

Reply:
Well, the author doesn't have to anything with the release step, that's something that maintainers do...that's why I kept it generic.

Reply:
SGTM.

Reply:
I'll rephrase this section to have a clearer separation of responsibilities. Contributors should be able to discern what they need to do and what maintainers will provide.